scripts/verify_privacy.sh

```
@@ -0,0 +1,567 @@
+
+monitor_connections_linux() {
+    local timeout=$1
+    local temp_file=$(mktemp)
+
+    print_info "Monitoring network connections (Linux)..."
+    print_info "Running test inference for ${timeout} seconds..."
+
+    # Start network monitoring in background
+    if command -v tcpdump &> /dev/null; then
+        # Use tcpdump (more comprehensive)
+        sudo timeout ${timeout} tcpdump -i any "host not 127.0.0.1 and host not ::1" -n 2>&1 > "$temp_file" &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)